src/contexts/auth/clearLocalStorageWithException.ts

```diff
@@ -1,6 +1,9 @@
 // Clears localStorage except for designated keys
 export default function clearLocalStorageWithException(exceptionKeys: string[]) {
-  const valuesToKeep = exceptionKeys.map((key) => [key, localStorage.getItem(key)]);
+  const valuesToKeep = [...exceptionKeys, 'GALLERY_ENABLE_MULTICHAIN_AUTH'].map((key) => [
+    key,
+    localStorage.getItem(key),
+  ]);
```

Comment:
reminder to remove this and replace with feature flags enabled for all employees

Reply:
waaait scratch that, the employee FF won't work bc it requires you to be logged in 😂 and this feature specifically requires you to be logged out!

let's just keep it localStorage. will write a note in the doc to remove this later